src/main.ts

```diff
@@ -1,13 +1,13 @@
 import { createApp } from 'vue';
-import routes from 'vite-plugin-pages/client';
+import routes from 'voie-pages';
 import { createRouter, createWebHistory } from 'vue-router';
 
 import { useMirageJS } from './plugins/miragejs';
 
 import './index.css';
 import App from './App.vue';
 
-const router = createRouter({
+export const router = createRouter({
```

Comment:
Router has own plugin file now

Reply:
Should get fixed when main is merged back into this branch.